src/clojure/tdlib_json/core.clj

```diff
@@ -1,48 +1,51 @@
 (ns tdlib-json.core
-  (:require [cheshire.core :as che]
-            [clojure.core.async :as async])
-  (:import [tdlib_json TgClient]))
+  (:require
+   [cheshire.core :as json]
+   [clojure.core.async :as async])
+  (:import [tdlib_json TgJsonClient]))
+
+
+(defn jsonify [messageMap]
+  (json/generate-string messageMap))
+
+(defn json-parse [string]
+  (json/parse-string string true))
 
-(def client (atom nil))
 
-(defn create-client [path-to-lib timeout]
-  (TgClient. path-to-lib timeout))
+(def client (atom nil))
 
 (def message-queue (atom nil))
 
-(defn jsonify [messageMap] (che/generate-string messageMap))
 
-(defn json-parse [string]
-  (che/parse-string string true))
+(defn create-client [path-to-lib verbosity]
+  (TgJsonClient. path-to-lib verbosity))
+
+(defn client-send [messageMap]
+  (.send @client (jsonify messageMap)))
```

Comment:
Lets remove cheshire in this PR. We would want to change this to always receive ^String.

Reply:
I think better use docstrings for this. Manual typehinting doesn't add some performance profit (no reflection warnings here).